BioXASAcquaman.pro

```diff
@@ -5,8 +5,9 @@
 TEMPLATE = subdirs
 SUBDIRS += \
 	Initialize.pro \
-	BioXASSideAcquaman.pro \
 	BioXASMainAcquaman.pro \
-	BioXASImagingAcquaman.pro \
+	BioXASMainTableCoordinator.pro \
```

Comment:
`BioXASMainTableCoordinator.pro` along with `BioXASSideAcquaman.pro` and `BioXASImagingAcquaman.pro` was original added in the `acquamanSuite.pro`. I think it will be better if we organize all the bioxas related pro files into `BioXASAcquaman.pro`.

or maybe we should move `BioXASMainTableCoordinator.pro` into `BioXASMainAcquaman.pro` and move `BioXASSideTableCoordinator.pro` into `BioXASSideAcquaman.pro`?